src/main/java/seedu/duke/entities/Order.java

```diff
@@ -0,0 +1,32 @@
+package seedu.duke.entities;
+
+import java.util.ArrayList;
+
+public class Order {
+    private ArrayList<Dish> order;
+
+    Order() {
+        this.order = new ArrayList<Dish>();
+    }
+
+    public void addDishToOrder(Dish dish) {
+        this.order.add(dish);
+    }
+
+    public void removeDishFromOrderByIndex(int i) {
+        this.order.remove(i);
```

Comment:
Maybe need to check whether the index is valid or not and throw an exception if invalid